man/systemctl.xml

```diff
@@ -876,6 +876,18 @@ kobject-uevent 1 systemd-udevd-kernel.socket systemd-udevd.service
             a similar filter for messages and might be more
             convenient.
             </para>
+            
+            <para>The Loaded: line in the output will show "loaded" if the unit has been loaded into memory and "not-found" if service can't be found.
+              Along with showing the path to the unit file, this line will also show the enablement state. 
+              Enabled commands start at boot. 
+              See the full table of possible enablement stats in documentation for the is-enabled command. </para>
+            <para>The Active: line shows active state. 
+              The value is usually "active" or "inactive". Active could mean started, bound, plugged in, etc depending on the unit type.
```

Comment:
@keszybz I disagree about applying the DRY principle to documentation even though it works well for code. Being strict about not repeating documentation can create a poor user experience where the user has to bounce around following a number of different references to get an answer when what they really wanted to do was to come to a single location to find their answer. I also doubt the definition of "Active" is going to change over much time and get so far out of sync.